### DISCUSSION TITLE
tunnel down after upgrading to 0.6

### DISCUSSION THREAD

**Discussion by sah66214** [Ysurac/openmptcprouter#3313] [General]
Hi , 
after i upgrading to 0.6 for both side my VPN tunnel is down , any idea ?
 it was work perfectly, before upgrade.
![Untitled](https://github.com/Ysurac/openmptcprouter/assets/50520976/173e9edc-5e5f-44bf-ac54-9ad5d0502375)

**Comment by Ysurac** [comment]
Check what you have in Status->System log. This should help to find why tunnel is down.

**Comment by sah66214** [comment]
sure , most error i see in these 2 days are same in below.thanks
Apr 26 08:09:54 OpenMPTCProuter daemon.err openvpn(omr)[13897]: TLS Error: TLS key negotiation failed to occur within 60 seconds (check your network connectivity)
Apr 26 08:09:54 OpenMPTCProuter daemon.err openvpn(omr)[13897]: TLS Error: TLS handshake failed
Apr 26 08:09:54 OpenMPTCProuter daemon.err openvpn(omr)[13897]: Fatal TLS error (check_tls_errors_co), restarting
Apr 26 08:09:54 OpenMPTCProuter daemon.notice openvpn(omr)[13897]: SIGUSR1[soft,tls-error] received, process restarting

**Reply by Ysurac** [reply]
I would need a bit more log.
You rebooted VPS after install ? There is no firewall on provider side ?

**Reply by sah66214** [reply]
[after reboot.txt](https://github.com/Ysurac/openmptcprouter/files/15131633/after.reboot.txt)
[before.txt](https://github.com/Ysurac/openmptcprouter/files/15131634/before.txt)
I attached 2 log text file . before and after a reboot router.
i rebooted VPS also ask provider , there is no restriction on firewall by default.
thanks in advance
